### PR TITLE
py3: adjust cache_timeout intervals

### PR DESCRIPTION
**Pull request by lasers**
This aims to make time intervals looks more consistent with user expectations. I changed it a bit to knock off an extra condition and favoring `seconds >= 0` first because that's what most modules uses. 

Addresses #1238.

### CODE REVIEW

**Comment by ultrabug** [comment]
Did you test this with pomodoro module? I did and when I click to pause/resume I see the seconds remaining get down and the time seems less consistent than master to me.

**Comment by ultrabug** [comment]
Yes @lasers you're right and that's what @tobes said too

But that's not what I meant in my comment. I said that it seems like your updated logic does something bad with the current module because when you click you loose a second on the timer (but you expect not to).

**Comment by lasers** [comment]
Is it not because there are no `prevent_refresh`... and you click it... Instant update right there? Especially during running state -> stopped state.

**Comment by ultrabug** [comment]
Possible, but can't see it on current time_in implementation

**Comment by lasers** [comment]
Sorry if I don't understand you.

For `pomodoro`? From what I gather, `0` or `1` should be same so we shouldn't have to touch anything. 

For other modules that refreshes every 5, 10, or 20 seconds... It's more obvious and can be hard to follow (i.e. I set 10, then I see easy-to-follow intervals versus random numbers all times) in songs or such... Always spitting out numbers that does not seems to be consistent / right.

**Comment by lasers** [comment]
Alternatively, we can close this PR and add `sync_to=0` to time-related modules.

**Comment by lasers** [comment]
My interpretation...  Seems like it rounds up. Maybe this is what you saw...
* `1547666333.0003157` - `0.1` = `1547666332.9003158` -->`sync_to=1` --> `1547666333.0`.

It might be fine 9/10 of the time... and on that 1/10, you get the above... no offsetting. And it may be mostly depending on  when you started/clicked pomodoro... then it'll sync to the nearest second soon enough... so it might be all about timing... mostly at startup / click.  Idk. 

Idk. One more issue left before we all are on same page...
